coordinator/src/db/user.rs

```diff
@@ -160,3 +161,22 @@ pub fn get_users(conn: &mut PgConnection, trader_ids: Vec<PublicKey>) -> Result<
 
     Ok(users)
 }
+
+pub fn check_version(conn: &mut PgConnection, trader_id: &PublicKey) -> Result<()> {
+    let user: User = users::table
+        .filter(users::pubkey.eq(trader_id.to_string()))
+        .first(conn)?;
+
+    let app_version = match user.version {
+        Some(version) => version,
+        None => bail!("No version found"),
```

Comment:
:+1: I think this makes sense. If the information is not there, we can assume that we are dealing with an old app.

Reply:
:wrench: We usually prefer to use `context()?` instead of `match` + `bail` when dealing with `Option`s.